control_software/src/hera_corr.py

```diff
@@ -815,7 +815,7 @@ def eq_initialize(self, host, verify=True):
                     failed.append((stream, (ant, pol)))
         if len(failed) > 0:
             raise RuntimeError('Failed to initialize EQ on %s: %s' %
-                               (host, ','.join(['%d=(%d%s)' % (ant,pol)
+                               (host, ','.join(['(%d%s)' % (ant,pol)
```

Comment:
The first %d looks like it should have been *stream*. Can you put it back and set it to stream?